config/services.yml

```diff
@@ -23,6 +23,32 @@ x-turbo-config: &default-turbo-config
 
 https://cmr.earthdata.nasa.gov:
 
+  - name: harmony/download
+    description: Service that performs no processing, it only returns download links for all of the matched granules from the CMR.
+    data_operation_version: '0.18.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/harmony/download
+    maximum_sync_granules: 0
+    capabilities:
+      subsetting:
+        bbox: false
+        variable: false
+        multiple_variable: false
+      reprojection: false
+      concatenation: false
+      all_collections: true
+      output_formats:
+        - application/netcdf # Incorrect mime-type, remove when no longer needed
+        - application/x-netcdf4
```

Comment:
What is with this output_formats? Would granules with formats other than netcdf be able to use this download service?

Reply:
There's a bit of an issue right now - we assume everything is in netcdf-4 format (which is valid for all of the datasets supported so far). In the future we need to know what the input format for a collection is to decide on whether format conversions are needed.

So this list is necessary to make sure that things asking for a format other than the original input format are passed on to a backend service to convert the format rather than providing the original download links.

Reply:
It makes sense. Thanks!